apps/client/src/common/api/ontimeApi.ts

```diff
@@ -143,15 +143,19 @@ export async function postOscSubscriptions(data: OscSubscription) {
 /**
  * @description HTTP request to download db in CSV format
  */
-export const downloadCSV = () => {
-  return fileDownload(ontimeURL, { name: 'rundown', type: 'csv' }, { type: 'text/csv;charset=utf-8;' });
+export const downloadCSV = (fileName?: string) => {
+  return fileDownload(ontimeURL, { name: fileName ?? 'rundown', type: 'csv' }, { type: 'text/csv;charset=utf-8;' });
```

Comment:
I made this new prop optional and added a fallback value of `rundown`, which was the default value prior to this change.
From a quick search it looks like this was being used in one other place so instead of adding the parameter as "rundown" I just added a fallback value in case the parameter isn't provided.

Should we just remove the "Export" option on the Sidebar?

Reply:
Thank you for cleaning up.

This makes sense to me. The export function in the `/cuesheet` doesnt have access to the project information. We also likely don't want to add it there so a fallback is good.

> Should we just remove the "Export" option on the Sidebar?

Yes please!